frontends/web/src/components/toast/Toast.jsx

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright 2018 Shift Devices AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from 'react';
+import style from './Toast.module.css';
+
+export class Toast extends Component {
+  state = {
+    active: false,
+  };
+
+  componentDidMount() {
+    setTimeout(this.show, 5);
+  }
+
+  show = () => {
+    this.setState({ active: true });
+  };
+
+  hide = () => {
+    this.setState({ active: false });
+    if (this.props.onHide) {
+      this.props.onHide();
+    }
+  };
+
+  render() {
+    const {
+      theme,
+      children,
+      withGuide,
+    } = this.props;
+    const { active } = this.state;
+    return (
+      <div
+        className={[style.toast, style[theme], active ? style.active : '', withGuide ? style.shifted : ''].join(' ')}>
+        <p>{children}</p>
+      </div>
+    );
+  }
+}
```

Comment:
accidentally pushed this while rebasing, ci didn't mind  no error. Seems tsconfig node moduleResolution just finds the toast module and doesn't care there there is potential conflict. Removing...

Reply:
PR udpated